Makefile

```diff
@@ -66,7 +66,7 @@ update-requirements: build
 	docker-compose run --rm api "cd /code && pip install pip-tools -U && pip-compile --upgrade requirements.in requirements-dev.in -o requirements-dev.txt && chmod a+r requirements-dev.txt"
 
 migrations: build
-	docker-compose run --rm api "./manage.py makemigrations"
+	docker-compose run --rm api "./manage.py makemigrations && ./manage.py migrate"
```

Comment:
This looks ok to me. My only concern would be that if you botch the migration somehow and you need to redo it, you will have to revert the changes or recreate the db. Then again, if people hit this flow and complain we can move it in a separate command.